### PR TITLE
Handled empty hostname in server-del command

### PR DESCRIPTION
**Pull request by Akasurde**
Fixes: https://fedorahosted.org/freeipa/ticket/6248

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

### CODE REVIEW

File: ipaclient/plugins/server.py [hunk 1]
```diff
@@ -12,6 +12,12 @@
 @register(override=True, no_fail=True)
 class server_del(MethodOverride):
     def interactive_prompt_callback(self, kw):
+        server_list = []
+        try:
+            server_list = kw['cn']
+        except Exception as e:
```
Comment (mbasti-rh) [inline ipaclient/plugins/server.py]: 1. Can you please use KeyError?
2. Please remove 'as e'. e is unused variable
3 you can use
```
server_list = kw.get('cn')
if not server_list:
    raise err......
```

File: ipaclient/plugins/server.py [hunk 2]
```diff
@@ -12,6 +12,12 @@
...
+            raise errors.ValidationError(name='cn', error=_(u'cn is required'))
```
Comment (stlaz) [inline ipaclient/plugins/server.py]: You should use RequirementError here.

File: ipaclient/plugins/server.py (hunk 1 above)
Comment (stlaz) [inline ipaclient/plugins/server.py]: Did you mean RequirementError?

File: ipaclient/plugins/server.py (hunk 2 above)
Comment (stlaz) [inline ipaclient/plugins/server.py]: You should use RequirementError here

**Comment by mbasti-rh** [comment]
NACK, see inline comments

**Comment by mbasti-rh** [comment]
Fixed upstream
master:
https://fedorahosted.org/freeipa/changeset/95a594af4c99255ea4da27e609cf41b79ca7ed91

**Comment by Akasurde** [comment]
@mbasti-rh @stlaz Thanks for comments

**Pull request closed without merging**